client/my-sites/plans/jetpack-plans/product-store/items-list/style-most-popular.scss

```diff
@@ -72,3 +72,30 @@
 	line-height: 24px;
 	font-weight: 600;
 }
+
+.featured-item-card--amount-saved {
+	margin-top: -4px;
+	margin-bottom: 8px;
+
+	.amount-saved--text {
+		font-weight: 600;
+		color: var(--studio-gray-70);
+		display: flex;
+		align-items: center;
+		gap: 4px;
+	}
+
```

Comment:
Not related to this PR, but I see a slight misalignment of the info button for the amount saved.

Can we fix that with this bit of styling here?

```suggestion

	.amount-saved--text button {
		display: flex;
	}

```

BEFORE
<img width="1225" alt="Screenshot 2022-12-19 at 11 49 02 AM" src="https://user-images.githubusercontent.com/18226415/208361075-3a1607a5-7a7e-4c83-a380-650b233d6fa0.png">

AFTER
<img width="1231" alt="Screenshot 2022-12-19 at 11 49 53 AM" src="https://user-images.githubusercontent.com/18226415/208361098-fce61d7c-e084-48e2-9d62-18785c740df2.png">

CC: @ilonagl